sdk/loadtestservice/azure-developer-loadtest/README.md

```diff
@@ -0,0 +1,161 @@
+
+# Azure Developer LoadTesting client library for Python
+Azure Developer LoadTesting provides client library in python to the user by which they can interact natively with Load Test Service.The service is for performing the load test to optimize application performance, scalability or capacity. The user can get the client-side and server-side metrices to see the details reported by the test engine and information about your Azure application components.
+
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install azure-developer-loadtesting
+```
+
+#### Prequisites
+
+- Python 3.7 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing Azure Developer LoadTesting instance.
+#### Create with an Azure Active Directory Credential
+To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
+As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`
+
+Use the returned token credential to authenticate the client:
+
+```python
+from azure.developer.loadtesting import LoadTestClient
+from azure.identity import DefaultAzureCredential
+client = LoadTestClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+```
+
+## Examples
+
+### Creating a load test 
+```python
+from azure.developer.loadtesting import LoadTestClient
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+client = LoadTestClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+try:
+    result = client.load_test_administration.create_or_update_test(
+        TEST_ID,
+        {
+            "resourceId": f"/subscriptions/{SUBSCRIPTION_ID}/resourceGroups/yashika-rg/providers/Microsoft.LoadTestService/loadtests/loadtestsdk",
```

Comment:
I guess you mean `SUBSCRIPTION_ID` is from environment variables but I think it is better to declare it before use it like:
`SUBSCRIPTION_ID = os.getenv("SUBSCRIPTION_ID ")`

Reply:
Resolved in https://github.com/Azure/azure-sdk-for-python/pull/25264/commits/faddc05c876a7926ac822e27fce970217dab4929